### PR TITLE
first part of azure docs

### PR DESCRIPTION
**Pull request by joeyouss**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the CODEOWNERS. 

## What Type of PR is This?

- [x] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screenshot.

### CODE REVIEW

**Review by Rohit-Kaliki — COMMENTED** [review]
Flow looks good, let's work on the language and other house keeping stuff.

File: docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+2. Select **Asset Governance**.
+3. Select **Enforcements**.
+4. Select **+ New Enforcement**.
+5. Enter a name for the enforcement.
+6. Optionally, enter a description of the enforcement.
+7. Select the cloud provider.
+8. Select the rules or rulesets that you want to enforce. You can use the **Search** box if you have multiple rules and are looking to enforce a particular rule or rule set.
+9. Select **Continue**. 
+10. Select the target accounts and target regions. You could select multiple accounts and regions.
+11. Set the frequency from **Hourly**, **Daily**, or **Weekly **options. In case you select Daily or Weekly, specify the day, time, and time zone to run the rule on schedule.
```
Comment (iamanmolm) [inline docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md:130]: Weekly is not properly formatted. "**Weekly** options"

File: docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+12. Toggle the **Dry Run** mode if you do not want to take action immediately.
+13. Select **Finish**. 
+
+    <DocImage path={require('./static/set-up-schedule.png')} width="60%" height="60%" title="Click to view full size image" />
+
+
+After setting up the schedule, you can view the enforcement on the **Rule Enforcements** page.
```
Comment (iamanmolm) [inline docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md:137]: "on the **Enforcements** page."

File: docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+6. Optionally, enter a description of the enforcement.
+7. Select the cloud provider.
+8. Select the rules or rulesets that you want to enforce. You can use the **Search** box if you have multiple rules and are looking to enforce a particular rule or rule set.
+9. Select **Continue**. 
+10. Select the target accounts and target regions. You could select multiple accounts and regions.
+11. Set the frequency from **Hourly**, **Daily**, or **Weekly **options. In case you select Daily or Weekly, specify the day, time, and time zone to run the rule on schedule.
+12. Toggle the **Dry Run** mode if you do not want to take action immediately.
+13. Select **Finish**. 
+
+    <DocImage path={require('./static/set-up-schedule.png')} width="60%" height="60%" title="Click to view full size image" />
```
Comment (iamanmolm) [inline docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md:134]: Screenshot is wrong. It should be of Enforcement List Page.

File: docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+
+
+After setting up the schedule, you can view the enforcement on the **Rule Enforcements** page. 
+
+<DocImage path={require('./static/view-rule-set.png')} width="60%" height="60%" title="Click to view full size image" />
+
+Furthermore, you can disable the enforcement at any time using the toggle button in the **Status** column. If you want to turn off the dry-run mode, select **Edit** from the vertical ellipsis menu (⋮) and switch to active mode.
```
Comment (iamanmolm) [inline docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md:141]: and then go to "Target And Schedule", use slider to turn off "Enforce Rule(s) in Dry Run mode" and click on Finish.

File: docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md [hunk 5]
```diff
@@ -0,0 +1,193 @@
...
+  
+<DocImage path={require('./static/rule-enforcements-page.png')} width="60%" height="60%" title="Click to view full size image" />
+
+### Update an Enforcement
+
+You can view any Enforcements on Rule Enforcements page. Expand the enforcement to view the rules, target accounts, and regions included in the enforcement. For updating, you can use the “Edit” option to update the enforcements as per your convenience.
```
Comment (iamanmolm) [inline docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md:147]: Remove "Expand the enforcement to view the rules, target accounts, and regions included in the enforcement." as this is incorrect. You cant expand the rule enforcement.

Comment (iamanmolm) [inline docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md:147]: "you can use the "Edit" button from the vertical ellipsis menu (⋮) to update the enforcements as per your convenience."

File: docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md [hunk 6]
```diff
@@ -0,0 +1,193 @@
...
+
+### Delete an Enforcement
+
+To delete an enforcement, simply click on “Delete”
```
Comment (iamanmolm) [inline docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md:151]: "To delete an enforcement, simply click on “Delete” from the vertical ellipsis menu (⋮)"

File: docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md [hunk 7]
```diff
@@ -0,0 +1,193 @@
...
+
+
+## Evaluations
+
+Evaluations include all the data about enforcements run (both, directly and from the test terminal). The Evaluations window also shows you the total cost impact with each enforcement i.e. the costs or spendings associated with each evaluation along with the last time that rule/rule set was enforced. With evaluations, you can view and audit all the enforcements and have access to all the historical data on your account.
```
Comment (iamanmolm) [inline docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md:156]: "run (both RUN ONCE from rule editor and from Enforcement)"

File: docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md [hunk 8]
```diff
@@ -0,0 +1,193 @@
...
+1. In your **Harness** application, go to **Cloud Costs**.
+2. Select **Asset Governance**.
+3. Select **Evaluations**.
+4. Select the rule to view the evaluation details. 
+The target accounts, regions, and evaluation logs are displayed.
+
+ <DocImage path={require('./static/asset-gov-eval.png')} width="60%" height="60%" title="Click to view full size image" />
+
+
+## Use filters in rule evaluation
```
Comment (iamanmolm) [inline docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md:170]: Make it subheading under Evaluations heading.

File: docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md (hunk 7 above)
Comment (iamanmolm) [inline docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md:156]: "you can view and audit all the enforcements that ran in the past."

File: docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md (hunk 8 above)
Comment (iamanmolm) [inline docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md:170]: Filters in Evaluations List Page.

File: docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md [hunk 9]
```diff
@@ -0,0 +1,193 @@
...
+4. Select one or more of the following criteria to filter the results:
+    * Rules
+    * Rule Sets
+    * Enforcements
+    * Minimum Cost Impact ($)
+    * AWS Filters
+
+      - Target Accounts
+      - Target Regions
+    * Azure Filters
```
Comment (iamanmolm) [inline docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md:186]: GCP Filters can also be added.

File: docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md [hunk 10]
```diff
@@ -0,0 +1,193 @@
...
+title: Harness Concepts
+description: This topic describes how to optimize cloud costs using asset governance.
+# sidebar_position: 2
+---
+
+
+# Harness Concepts: Rules, Rule Sets, Enforcements and Evaluations to optimise cloud costs
+
+:::note
+Currently, GCP support for Asset Governance is behind the feature flag **CCM_ENABLE_GCP_CLOUD_ASSET_GOVERNANCE_UI** Contact [Harness Support](mailto:support@harness.io) to enable the feature.
```
[the remark this answers is not in the document]
Comment (joeyouss) [reply docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md:11]: done

File: docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md [hunk 11]
```diff
@@ -0,0 +1,193 @@
...
+:::
+
+## Rules
+
+Rules help you set up asset governance for your cloud provider. A rule is essentially a small file with a set of logic that you can run on your cloud infrastructure. For example, there might be a scenario in which you want to migrate your Amazon EBS volumes from GP2 to GP3. In such a case, we will write a small file which will run this logic for us.
```
[the remark this answers is not in the document]
Comment (joeyouss) [reply docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md:16]: done

File: docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md [hunk 12]
```diff
@@ -0,0 +1,193 @@
...
+
+:::note
+Currently, GCP support for Asset Governance is behind the feature flag **CCM_ENABLE_GCP_CLOUD_ASSET_GOVERNANCE_UI** Contact [Harness Support](mailto:support@harness.io) to enable the feature.
+:::
+
+## Rules
+
+Rules help you set up asset governance for your cloud provider. A rule is essentially a small file with a set of logic that you can run on your cloud infrastructure. For example, there might be a scenario in which you want to migrate your Amazon EBS volumes from GP2 to GP3. In such a case, we will write a small file which will run this logic for us. 
+
+Ideally, rules include policy, resource filters, and actions.
```
[the remark this answers is not in the document]
Comment (joeyouss) [reply docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md:18]: done

File: docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md [hunk 13]
```diff
@@ -0,0 +1,193 @@
...
+
+- A **policy** is defined in YAML format and consists of filters and actions that are applied to a specific type of cloud resource set up by you.
```
[the remark this answers is not in the document]
Comment (joeyouss) [reply docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md:20]: ok

File: docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md [hunk 14]
```diff
@@ -0,0 +1,193 @@
...
+
+- A **resource** is the type of cloud resource or service on which the rule will be run or to which the actions and filters will be applied, such as AWS EC2, AWS S3, GCP SQL, or Azure VMs.
```
[the remark this answers is not in the document]
Comment (joeyouss) [reply docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md:22]: done

File: docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md (hunk 7 above)
Comment (joeyouss) [reply docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md:156]: done

File: docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md [hunk 15]
```diff
@@ -0,0 +1,193 @@
...
+
+- A **filter**, as the name suggests, is a criteria used to narrow down the results based on the attributes. These attributes can include anything such as tags, metadata, or any other resource property provided by you. When the filter is applied, only those resources that match the criteria specified in the filter are given as a result.
+
+- **Actions** are operations that have to be performed on the resource and are applied to the output of the filters specified in the policy when the rule is run. Actions can include things like terminating an EC2 instance, deleting an S3 bucket, or sending an email notification.
```
[the remark this answers is not in the document]
Comment (joeyouss) [reply docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md:26]: done

File: docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md [hunk 16]
```diff
@@ -0,0 +1,193 @@
...
+
+So essentially, a rule is a file that includes logic defined by a policy that performs certain actions on the resource based on the filters provided by the user. Rules can include multiple policies, and policies include resource filters and actions.
```
[the remark this answers is not in the document]
Comment (joeyouss) [reply docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md:28]: done

File: docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md [hunk 17]
```diff
@@ -0,0 +1,193 @@
...
+9. Enter the YAML policy in the rule editor.
+10. Select **Save**. If the policy is invalid, an error message is displayed.
+10. Select the **Account** and the **Region** from the dropdown list in the Test Terminal.
+11. Select **Dry Run** to view the instances or services that will be acted upon when you enforce the rule.
+12. After evaluating the output, select **Run Once** to execute the rule. 
+
+  <DocImage path={require('./static/asset-governance-rule-enforcement.png')} width="60%" height="60%" title="Click to view full size image" />
+
+:::important note
+Harness provides some out-of-the-box policies for EC2, RDS, EBS, ELB, and S3 that can be enforced. These policies cannot be edited but can be cloned.
```
[the remark this answers is not in the document]
Comment (joeyouss) [reply docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md:58]: done

File: docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md [hunk 18]
```diff
@@ -0,0 +1,193 @@
...
+:::
+
+## Rule Sets
+
+As mentioned previously, a Rule can have multiple policies. However, when there are multiple rules with multiple policies, it can become hard to manage them all together. This is where **Rule Sets** come into the picture. Rule sets serve as logical bindings on top of individual rules that help you organize and manage rules. By organizing rules into sets, organizations improve accessibility and simplify maintenance, as enforcements can be made against the entire rule set rather than individual rules.
```
[the remark this answers is not in the document]
Comment (joeyouss) [reply docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md:63]: done

File: docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md [hunk 19]
```diff
@@ -0,0 +1,193 @@
...
+
+### CRUD operations on Rule Sets
+### Create a new Rule Set
+
+To create a rule set, perform the following steps:
+
+1. In **Harness**, go to **Cloud Costs**.
+2. Select **Asset Governance**.
+3. Select **Rules**.
+4. Select the **Rule sets** tab.
```
[the remark this answers is not in the document]
Comment (joeyouss) [reply docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md:75]: done

File: docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md [hunk 20]
```diff
@@ -0,0 +1,193 @@
...
+5. Select **+ New Rule Set**.
+6. Enter a name for the rule set.
+7. Optionally, enter a description of the rule set.
+8. Select the cloud provider.
+9. Select the rules that you want to enforce.
```
[the remark this answers is not in the document]
Comment (joeyouss) [reply docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md:80]: done

File: docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md [hunk 21]
```diff
@@ -0,0 +1,193 @@
...
+### Update a Rule Set
+
+You can view the Rule Set on the Asset Governance Rules page. Expand the rule set to view the individual rules in the rule set. You can click on individual rules to update them and/or use the “Edit” option to update the entire rule set.
+
+### Delete a Rule Set
+To delete a Rule Set, simply click on the “Delete” option.
+
+## Enforcements
+
+Enforcements enable you to enforce a certain set of rules or rule sets (also known as governance guardrails) against a specific set of targets (accounts, subscriptions, and projects) to run periodically. Sometimes, we need rules to run periodically, such as every day, week, or month. However, running these rules manually every day or week at a specified time creates extra overhead and is a slow process prone to manual errors. To solve this, enforcements allow users to set up a timely schedule and choose the day, time, and frequency at which they want their rules or rule sets to be enforced.
```
[the remark this answers is not in the document]
Comment (joeyouss) [reply docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md:100]: this was added by Rohit, change it?

[the remark this answers is not in the document]
Comment (joeyouss) [reply docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md:100]: done

[the remark this answers is not in the document]
Comment (joeyouss) [reply docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md:100]: done

File: docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md [hunk 22]
```diff
@@ -0,0 +1,193 @@
...
+
+For example, a user can create an enforcement to schedule the deletion of all unused EC2 instances older than 60 days. This enforcement will run on the **days specified by the user**, at the **specified time**, and with the **specified frequency (hourly, daily, monthly**). For instance, you could set it to run daily at 2:00 AM to ensure that any EC2 instances meeting the criteria are removed. Alternatively, you might choose to run it hourly during peak usage times, or monthly for less critical cleanup tasks. 
+
+While setting up a new enforcement, you can select the following:
+- Cloud provider: Currently we support GCP, Azure and AWS
+Rule Sets: You can select the Rule Sets that your Enforcement will consist of.
```
[the remark this answers is not in the document]
Comment (joeyouss) [reply docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md:106]: done

[the remark this answers is not in the document]
Comment (joeyouss) [reply docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md:106]: done

File: docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md (hunk 1 above)
Comment (joeyouss) [reply docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md:130]: done

File: docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md (hunk 2 above)
Comment (joeyouss) [reply docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md:137]: done

File: docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md [hunk 23]
```diff
@@ -0,0 +1,193 @@
...
+- **Actions** are operations that have to be performed on the resource and are applied to the output of the filters specified in the policy when the rule is run. Actions can include things like terminating an EC2 instance, deleting an S3 bucket, or sending an email notification.
+
+So essentially, a rule is a file that includes logic defined by a policy that performs certain actions on the resource based on the filters provided by the user. Rules can include multiple policies, and policies include resource filters and actions.
+
+<DocImage path={require('./static/anatomy_of_a_rule.png')} width="70%" height="70%" title="Click to view full size image" />
+
+<DocImage path={require('./static/rule_example.png')} width="80%" height="80%" title="Click to view full size image" />
+
+### CRUD operations on a Rule
+### Create a new rule
```
[the remark this answers is not in the document]
Comment (joeyouss) [reply docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md:35]: done

File: docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md [hunk 24]
```diff
@@ -0,0 +1,193 @@
...
+
+:::important note
+Harness provides some out-of-the-box policies for EC2, RDS, EBS, ELB, and S3 that can be enforced. These policies cannot be edited but can be cloned.
+:::
+
+## Rule Sets
+
+As mentioned previously, a Rule can have multiple policies. However, when there are multiple rules with multiple policies, it can become hard to manage them all together. This is where **Rule Sets** come into the picture. Rule sets serve as logical bindings on top of individual rules that help you organize and manage rules. By organizing rules into sets, organizations improve accessibility and simplify maintenance, as enforcements can be made against the entire rule set rather than individual rules.
+
+  <DocImage path={require('./static/rule_set.png')} width="90%" height="90%" title="Click to view full size image" />
```
[the remark this answers is not in the document]
Comment (joeyouss) [reply docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md:65]: don e

File: docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md [hunk 25]
```diff
@@ -0,0 +1,193 @@
...
+
+10. Select **Create Rule Set**. 
+The rule set is created successfully. You can view the rule set on the **Asset Governance Rules** page. Expand the rule set to view the individual rules in the rule set.
+
+  <DocImage path={require('./static/view-rule-set.png')} width="60%" height="60%" title="Click to view full size image" />
+
+11. Select **Enforce Rule Set** in the Enforcements column to enforce this rule set.
+### Update a Rule Set
+
+You can view the Rule Set on the Asset Governance Rules page. Expand the rule set to view the individual rules in the rule set. You can click on individual rules to update them and/or use the “Edit” option to update the entire rule set.
```
[the remark this answers is not in the document]
Comment (joeyouss) [reply docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md:93]: done

File: docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md (hunk 4 above)
Comment (joeyouss) [reply docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md:141]: done

File: docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md (hunk 5 above)
Comment (joeyouss) [reply docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md:147]: done

Comment (joeyouss) [reply docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md:147]: done

File: docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md (hunk 6 above)
Comment (joeyouss) [reply docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md:151]: done

File: docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md (hunk 7 above)
Comment (joeyouss) [reply docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md]: done

File: docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md (hunk 8 above)
Comment (joeyouss) [reply docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md:170]: done

Comment (joeyouss) [reply docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/2-Harness-Concepts.md:170]: done

**Review by iamanmolm — CHANGES_REQUESTED** [review]
Dummy

**Review by iamanmolm — APPROVED** [review]

**Pull request merged**